sources/packages/backend/apps/db-migrations/src/sql/Notifications/Create-notifications.sql

```diff
@@ -0,0 +1,42 @@
+CREATE TABLE IF NOT EXISTS sims.notifications (
+    id SERIAL PRIMARY KEY,
+    -- Reference Columns
+    user_id INT REFERENCES users(id),
+    template_id UUID NOT NULL,
+    message_id INT REFERENCES messages(id) ON DELETE CASCADE,
+    message_payload jsonb NOT NULL,
```

Comment:
What is the purpose of message payload ? is it not based on the message id? if yes it can be moved to the messages table.

Reply:
It is not based on message id. It is the same payload that is passed as a parameter to the GC Notification service. Message id is reference to messages table (soon to be change to notification_messages).